quests/argeas/bandit.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<quests>
+  <its:rules xmlns:its="http://www.w3.org/2005/11/its" version="1.0">
+    <its:translateRule selector="//var/quest/@name" translate="yes"/>
+    <its:translateRule selector="//var/quest/name" translate="yes"/>
+    <its:translateRule selector="//var/quest/text" translate="yes"/>
+    <its:translateRule selector="//*" translate="no"/>
+  </its:rules>
+  <var id="26">
+    <effect map="009-2.tmx" npc="182" value="1" effect="45"/>
+    <effect map="009-2.tmx" npc="182" value="2,6" effect="46"/>
```

Comment:
quests that are done should not display any effect on the npc, so here you should not have state `6`